modules/installation-images-samples-disconnected-mirroring-assist.adoc

```diff
@@ -11,7 +11,11 @@ During installation, {product-title} creates a config map named `imagestreamtag-
 
 The format of the key for each entry in the data field in the config map is `<image_stream_name>_<image_stream_tag_name>`.
 
-During a disconnected installation of {product-title}, the status of the Cluster Samples Operator is set to `Removed`. If you choose to change it to `Managed`, it installs samples. 
+During a disconnected installation of {product-title}, the status of the Cluster Samples Operator is set to `Removed`. If you choose to change it to `Managed`, it installs samples.
+[NOTE]
+====
+The use of clusters in a restricted or discontinued network may require actions in services external to your network. Some example services include: Github, RubyGems, PyPi, and others. There might be additional steps to take that allow the cluster samples operators's objects obtain the services they require.
```

Comment:
I find the phrasing of "the use of clusters" awkward. The customer is using _samples_ in a network-restricted or air-gapped environment. Proposing the following:

```suggestion
The use of samples in a network-restricted or disconnected environment may require access to services outside the network. Some example services include: Github, Maven Central, npm, and others. There might be additional steps to take that allow the cluster samples operators's objects to reach the services they require.
```

Also recommending we reference Maven Central and npm - Java and NodeJS are commonly used by our customers, and we have often had asks for specific support in these languages.

Reply:
@adambkaplan Would RubyGems or PyPi still be applicable here or should they just be replaced?

Reply:
They would also be applicable - any external service that provides dependencies.